container-images/tcib/base/os/tobiko/tobiko.yaml

```diff
@@ -7,10 +7,8 @@ tcib_actions:
 - run: chmod 440 /etc/sudoers.d/tobiko_sudoers
 - run: mkdir -p /var/lib/tempest/external_files
 - run: chown -R tobiko.tobiko /var/lib/tobiko
-- run: git clone https://opendev.org/x/tobiko /var/lib/tobiko
-  # The following line doesn't work
-  #- run: cat {{ tobiko_config }} >> /var/lib/tobiko/tobiko.conf
-- run: python3 -m pip install --upgrade pip tox
+- run: python3 -m pip install --upgrade pip
+- run: python3 -m pip install 'tox>=3.8,<4.0'
```

Comment:
Was about to question this but noticed https://opendev.org/x/tobiko/src/branch/master/roles/tobiko-ensure-tox/defaults/main.yaml